pc2cps

```diff
@@ -201,8 +201,8 @@ def main(args):
         'Default-Components': [':%s' % component_name],
     }
     extend(spec, 'Version', package.attr.get('Version'))
-    extend(spec, 'Description', package.attr.get('Description'))
```

Comment:
Both `Description` and `Website` are part of the [Supplemental Schema](https://mwoehlke.github.io/cps/schema-supplement.html). It may be that the reader needs to preserve and allow access to unknown fields in addition to explicit extensions, but replacing these with pkgconfig-specific extensions is the wrong thing to do.

Reply:
How is the supplemental schema used in `cps.py`? When I read a CPS file that contains these fields, they seem to be ignored and discarded. I don't see them in the `components` variable.

Reply:
It isn't, presently. Hence my comment: "it may be that the reader needs to preserve and allow access to unknown fields in addition to explicit extensions"...

I haven't thought about what form that should take; considering we would be adding such a feature to handle stuff from the supplemental schema, it probably should not be overly obtuse.